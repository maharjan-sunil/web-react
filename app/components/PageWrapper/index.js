import styled from 'styled-components';

const PageWrapper = styled.div`
  padding-top: 15px;
  padding-bottom: 15px;
  padding-right: 15px;
  padding-left: 15px;
  margin-right: auto;
  margin-left: auto;
`;

export const DetailWrapper = styled.div`
  background: #fff;
  border: 1px solid $gray-300;
  margin-bottom: 30px;
  position: relative;
`;

export default PageWrapper;
