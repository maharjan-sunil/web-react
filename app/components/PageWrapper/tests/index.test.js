import React from 'react';
import { render } from 'react-testing-library';

import PageWrapper from '../index';

describe('<PageWrapper />', () => {
  it('should render an <PageWrapper> tag', () => {
    const { container } = render(<PageWrapper />);
    const element = container.firstElementChild;
    expect(element.tagName).toEqual('DIV');
  });

  it('should match snapshot ', () => {
    const { container } = render(<PageWrapper />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
