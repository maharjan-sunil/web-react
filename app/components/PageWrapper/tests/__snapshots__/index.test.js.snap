// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageWrapper /> should match snapshot  1`] = `
<div
  class="PageWrapper-az8zmc-0 ljdZGq"
/>
`;
